### PR TITLE
fix: corrupted paper-wallet backup

### PR DESCRIPTION
**Pull request by OlegMakarenko**
**What the current behaviour is**
The paper wallet lib uses 2 encoded PDF docs. The first one contains Mnemonic page, the second one -Account page. The output of Paper Wallet lib is a single PDF doc which always contains 1 Mnemonic page and Account pages depending on the number of `accountInfos` passed to the constructor. To do it the paper wallet lib uses the "Mnemonic" PDF as a base and copies pages from "Account" PDF.

**Why this is an issue.**
There is an issue with the `copyPages()` method form "pdf-lib" which is used to copy Account page from one PDF doc to another. Copied page is corrupted.

image

**What's the fix**
This PR changes the way how the Account page is cloning. Instead of coping Account page from separate (foreign) PDF doc, now Mnemonic and Account pages are already in a single doc. It lets to avoid using a `copyPages()` method form "pdf-lib" which introduces an issue.

### CODE REVIEW

**Comment by 0x6861746366574** [comment]
@yilmazbahadir and @AnthonyLaw to review. Please make sure to fix any spelling errors throughout the commit. 

It's not clear _what_ the issue was originally and how this fixes it. Could you extend the original issue following the format [seen here](https://github.com/symbol/statistics-service/pull/121)? 

As a rule of thumb, you want to describe these three things:

* **What** the current behaviour is.
* **Why** this is an issue.
* **How** you changed this behaviour.

**Comment by AnthonyLaw** [comment]
The code looks good, but 2 problems for me.
1. I have a problem with `npm i` in the packages/paper-wallets folder.
2. how to run the code?

**Comment by yilmazbahadir** [comment]
> The code looks good, but 2 problems for me.
> 
> 1. I have a problem with `npm i` in the packages/paper-wallets folder.
> 2. how to run the code?

I think this addresses the same question with a suggestion => https://github.com/symbol/wallets-lib/pull/5#issuecomment-1007384072

File: packages/paper-wallets/index.ts [hunk 1]
```diff
@@ -98,17 +97,39 @@ class SymbolPaperWallet {
    * Exports as a PDF Uin8Array
    */
   async toPdf(): Promise<Uint8Array> {
+    // Load teplate pdf document. It consists of 2 pages - mnemonic and account.
```
Comment (yilmazbahadir) [inline packages/paper-wallets/index.ts]:
```suggestion
    // Load template pdf document. It consists of 2 pages - mnemonic and account.
```

**Review by yilmazbahadir — COMMENTED** [review]
Can you run `lint:fix` in the whole project before you merge this one? (after the second PR merged ofc)

**Comment by OlegMakarenko** [comment]
@0x6861746366574 description updated.

**Comment by OlegMakarenko** [comment]
@AnthonyLaw 

> I have a problem with npm i in the packages/paper-wallets folder.

What's the issue? Can you please provide a console log.

> how to run the code?

You can build it locally and follow these instructions:
https://github.com/symbol/wallets-lib/tree/main/packages/paper-wallets

**Comment by AnthonyLaw** [comment]
> @AnthonyLaw
> 
> > I have a problem with npm i in the packages/paper-wallets folder.
> 
> What's the issue? Can you please provide a console log.
> 
> > how to run the code?
> 
> You can build it locally and follow these instructions: https://github.com/symbol/wallets-lib/tree/main/packages/paper-wallets

I think the problem caused by the M1 chip.
image

**Comment by OlegMakarenko** [comment]
@AnthonyLaw can you please open a new issue regarding this?

**Comment by 0x6861746366574** [comment]
Oops. Tried to rename the branch and forgot PR's are very particular. 😁

**Review by yilmazbahadir — COMMENTED** [review]
Deleting `package-lock.json`?

**Comment by OlegMakarenko** [comment]
> Deleting package-lock.json?

@yilmazbahadir yes. This is the old one from `packages/paper-wallets/`. 
We still have `package-lock.json` in the project root folder

**Comment by yilmazbahadir** [comment]
> > Deleting package-lock.json?
> 
> @yilmazbahadir yes. This is the old one from `packages/paper-wallets/`. We still have `package-lock.json` in the project root folder :)

~~Doesn't seem right to me if we want to turn this into a monorepo, I think every package should have its own package.json otherwise it wouldn't be a package, right? I think we haven't decided the tool for the monorepo yet, but it could be `lerna` or `bazel` or what the team has decided to use in `symbol` monorepo.~~


Just to verify @0x6861746366574, do we want to bundle the dependencies together and release them as a single package(wallets-lib) or do we want to turn this into a monorepo and release it as separate packages in the near future? Stuffing sounds like it is the former(single package) from your [discord message](https://discord.com/channels/856325968096133191/865180563433979904/928666762357137488) just wanted to verify before we proceed.

**Comment by 0x6861746366574** [comment]
> > > Deleting package-lock.json?
> > 
> > 
> > @yilmazbahadir yes. This is the old one from `packages/paper-wallets/`. We still have `package-lock.json` in the project root folder :)
> 
> ~Doesn't seem right to me if we want to turn this into a monorepo, I think every package should have its own package.json otherwise it wouldn't be a package, right? I think we haven't decided the tool for the monorepo yet, but it could be `lerna` or `bazel` or what the team has decided to use in `symbol` monorepo.~
> 
> Just to verify @0x6861746366574, do we want to bundle the dependencies together and release them as a single package(wallets-lib) or do we want to turn this into a monorepo and release it as separate packages in the near future? Stuffing sounds like it is the former(single package) from your [discord message](https://discord.com/channels/856325968096133191/865180563433979904/928666762357137488) just wanted to verify before we proceed.

Correct, we want to package them as a single library in the interim. 
Once we do the rewrite, the QR library and HD wallet will be deprecated (as this can be handled by the SDK). The address book should simply be a module of the headless client.

**Pull request merged**